src/diff.js

```diff
@@ -1,5 +1,13 @@
 const parseDiff = require( 'parse-diff' );
 
+/**
+ * Fetch which files are different in a particular push changeset.
```

Comment:
This is actually about mapping the lines in our files to lines in the diff; we used to use it for the inline comments before GH checks existed, and it's what generates the "x errors, but none on lines in this PR" (which is broken)

```suggestion
 * Work out which files and lines are contained in this PR.
```